config/config_server.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"time"
+)
+
+type ServerConfig struct {
+	Version   Version         `mapstructure:"version"`
+	Log       LogConfig       `mapstructure:"log"`
+	Serve     Serve           `mapstructure:"serve"`
+	Scheduler SchedulerConfig `mapstructure:"scheduler"`
+	Telemetry TelemetryConfig `mapstructure:"telemetry"`
+}
+
+type Serve struct {
+	Port                int           `mapstructure:"port" default:"9100"`    // port to listen on
+	Host                string        `mapstructure:"host" default:"0.0.0.0"` // the network interface to listen on
+	IngressHost         string        `mapstructure:"ingress_host"`           // service ingress host for jobs to communicate back to optimus
+	AppKey              string        `mapstructure:"app_key"`                // random 32 character hash used for encrypting secrets
+	DB                  DBConfig      `mapstructure:"db"`
+	ReplayNumWorkers    int           `mapstructure:"replay_num_workers" default:"1"`
+	ReplayWorkerTimeout time.Duration `mapstructure:"replay_worker_timeout" default:"120s"`
+	ReplayRunTimeout    time.Duration `mapstructure:"replay_run_timeout"`
```

Comment:
Create a separate struct for Replay config and move these fields. They are logically related to each other.